pkg/console/operator/sync_v400.go

```diff
@@ -105,16 +170,27 @@ func sync_v400(co *consoleOperator, operatorConfig *operatorv1.Console, consoleC
 		logrus.Printf("\t deployment changed: %v", depChanged)
 	}()
 
-	// at this point there should be no existing errors, we survived the sync loop
-	// pass back config (updated), and bool indicating change happened so we can update
-	// the cluster operator status
-	return operatorConfig, consoleConfig, toUpdate, nil
+	return nil, nil, toUpdate, nil
 }
 
-func SyncConsoleConfig(co *consoleOperator, consoleConfig *configv1.Console, route *routev1.Route) (*configv1.Console, error) {
-	logrus.Printf("Updating console.config.openshift.io with hostname: %v \n", route.Spec.Host)
-	consoleConfig.Status.PublicHostname = util.HTTPS(route.Spec.Host)
-	return co.consoleConfigClient.UpdateStatus(consoleConfig)
+func handleSyncErrorCondition(operatorConfig *operatorv1.Console, message string) {
```

Comment:
So if you call this & the `condition` is added, but the `handleSyncErrorCondition` doesn't actually do an `Update()` request, its going to lose this change.